### DISCUSSION TITLE
How to set keyword blocking.

### DISCUSSION THREAD

**Discussion by davros365** [proginosko/LeechBlockNG#476] [Q&A] [answered]
I've tried to block the keyword "browser", to stop new browsers from being downloaded. I entered * and then on the next line ~browser but it doesn't work. I've also tried it with a capital B. This is how it is shown on the FAQs. and I've entered keywords this way before and it worked.
What am I doing wrong?

**Comment by proginosko** [comment]
Please export your options to a text file (Options > General > Export Options) and email it to me at leechblockng@proginosko.com so I can take a closer look.

**Reply by davros365** [reply]
Thanks for your response. I sent that off on August 1st. 
Leechblock is now blocking almost everything. The indications are that the block with * ~Browser ~Tor in it, is blocking anything with the word "Leechblock", most emails and even this forum. I'm only able to access this forum now because I removed Leechblock from Chrome. I had to remove it from Mozilla and ms Edge as well, as it was blocking almost everything on all 3 browsers.

**Reply by proginosko** [reply, marked as answer]
Sorry, your email ended up in spam.

The problem with your settings is that in Block Set 1, you have "https://" in front of all your site names. You need to remove that, and simplify the list by only including the domain names.

Your second block set is not having any effect because you've selected "within time periods AND after time limit" but you haven't specified a time limit. See here: https://www.proginosko.com/leechblock/faq/difference-or-and/

I recommend you read the examples page to get a better handle on the settings: https://www.proginosko.com/leechblock/examples/

**Reply by davros365** [reply]
Thanks, I didn't notice that, so the other block sets were correct then and must have been doing the blocking? To recap, I'm trying to stop new browsers from being downloaded. I've now wiped all blocks clean and only have block set 1 with * ~Browser in it, however, the word Browser appears often and the result is it's blocking too many things. I tried before (in the text file I sent you) to block all browser websites but there are just too many and it was easy to bypass by finding a new download site. Do you have a suggestion of what setting/keywords I can use to stop new browsers from being downloaded?

**Reply by proginosko** [reply]
Yeah, blocking by keyword probably isn't the solution. I see that you've been trying to block sites where you can download Brave. In that case, you could use a wildcard block like this:

`**brave`

That will block any page with 'brave' in the URL. You could do something similar for other browser names.

**Reply by davros365** [reply]
Ah, so that won't block sites with brave in the site text, only when brave is in the url! I'll give that a try, thanks.

**Reply by proginosko** [reply]
Correct.
